R/pkg/R/mllib_clustering.R

```diff
@@ -610,3 +616,59 @@ setMethod("write.ml", signature(object = "LDAModel", path = "character"),
           function(object, path, overwrite = FALSE) {
             write_internal(object, path, overwrite)
           })
+
+#' PowerIterationClustering
+#'
+#' A scalable graph clustering algorithm. Users can call \code{spark.assignClusters} to
+#' return a cluster assignment for each input vertex.
+#'
```

Comment:
remove empty line - empty is significant in roxygen2

Reply:
I'll be more careful about this roxygen2 stuff.